README.md

```diff
@@ -99,6 +99,27 @@ New var:....... upper = 832
 74 453.0 832
 New var:....... mid = 453.0
 09:37:35.882486 line        13         print(lower, mid, upper)
+Total elapsed time: 00:00:00.000344
+```
+
+If `elapsed_time` is `True`, print elapsed time format.
+
+```python
+import pysnooper
+import random
+
+def foo(elapsed_time=False):
+    lst = []
+    for i in range(10):
+        lst.append(random.randrange(1, 1000))
+
+    with pysnooper.snoop(elapsed_time=elapsed_time):
```

Comment:
Just write `with pysnooper.snoop(elapsed_time=True):` and keep the example simple. No good reason to pass arguments around.

This particular code was used to demonstrate the advantage of `with` - in particular, nothing from the for loop above appears in the output. Use different code that is well suited for demonstrating the usefulness of `elapsed_time`. It can be much shorter and should probably have some sleeps in it.

Show the output of `elapsed_time=True`, and explain what the difference is. There will always be a line at the end `Total elapsed time:` even with `elapsed_time=False` - that is already confusing and potentially even undesirable. If that's not going to change, at least make it clear in documentation. People will easily miss that the numbers in the time column have changed with `elapsed_time=True`.

All of this should be much lower down anyway, probably under advanced usage. This section of the README is for the basics.

Reply:
I'll fix this later